config.json

```diff
@@ -5,7 +5,7 @@
   "platforms": {
     "scss": {
       "transformGroup": "scss",
-      "buildPath": "./src/scss/sd/",
+      "buildPath": "scss/sd/",
```

Comment:
¿Por qué ha cambiado la ruta, @fdzsergio?

Reply:
El resto de plataformas tienen un path más descriptivo en el nombre, y al generar una carpeta `src` pues no se sabe muy bien para que plataforma se está creando. Además no está agregada en el `.gitignore` y se puede subir por equivocación. También me cercioré que en el proyecto de `maia` se estaba sobreescribiendo la config de este proyecto en el [nuxt.config.js](https://github.com/NextChance/maia/blob/dev/nuxt.config.js). Yo incluso lo que propondría sería cambiar la config de aquí por la que tenéis en ese repo. ¿Como lo veis? así quedaría un par de lineas en vuestro config.

#### maia/nuxt.config.js
```javascript
const StyleDictionary = require('style-dictionary').extend('config.json');
StyleDictionary.buildPlatform('scss');
```

#### chatme-style-dictionary/config.json
```suggestion
      "buildPath": "./assets/scss/sd/",
```